### PR TITLE
Updating Lab api and client secret

### PR DESCRIPTION
**Pull request by trwalke**
# Updating Lab api and client secret

- [ ] You've read the [Contributor Guide](https://github.com/AzureAD/microsoft-identity-web/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/AzureAD/microsoft-identity-web/blob/master/CODE_OF_CONDUCT.md).
- [ ] You've included unit or integration tests for your change, where applicable.
- [ ] You've included inline docs for your change, where applicable.
- [ ] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

Summary of the changes (Less than 80 chars)

## Description

{Detail}

Fixes #{bug number} (in this specific format)

### CODE REVIEW

File: tests/Microsoft.Identity.Web.Test.Common/TestConstants.cs [hunk 1]
```diff
@@ -105,17 +105,15 @@ public static class TestConstants
...
-        public const string ConfidentialClientLabTenant = "72f988bf-86f1-41af-91ab-2d7cd011db47";
+        public const string ConfidentialClientId = "88f91eac-c606-4c67-a0e2-a5e8a186854f";
+        public const string ConfidentialClientLabTenant = "f645ad92-e38d-4d1a-b510-d1b09a74a8ca";
         public const string OBOUser = "fIDLAB@msidlab4.com";
         public const string OBOClientSideClientId = "c0485386-1e9a-4663-bc96-7ab30656de7f";
         public static string[] s_oBOApiScope = new string[] { "api://f4aa5217-e87c-42b2-82af-5624dd14ee72/.default" };
         public const string LabClientId = "f62c5ae3-bf3a-4af5-afa8-a68b800396e9";
         public const string MSIDLabLabKeyVaultName = "https://msidlabs.vault.azure.net";
-        public const string AzureADIdentityDivisionTestAgentSecret = "MSIDLABAccessSecret";
+        public const string AzureADIdentityDivisionTestAgentSecret = "MSIDLAB4-IDLABS-APP-AzureADMyOrg-CC";
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test.Common/TestConstants.cs:115]: Why do we need that?

File: tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs [hunk 2]
```diff
@@ -49,7 +49,7 @@ public class AcquireTokenForAppIntegrationTests
         {
             _output = output;
 
-            KeyVaultSecretsProvider keyVaultSecretsProvider = new KeyVaultSecretsProvider(TestConstants.BuildAutomationKeyVaultName);
+            KeyVaultSecretsProvider keyVaultSecretsProvider = new KeyVaultSecretsProvider(TestConstants.MSIDLabLabKeyVaultName);
```
Comment (jmprieur) [inline tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs:52]: Why is this needed?

File: tests/Microsoft.Identity.Web.Test.Common/TestConstants.cs (hunk 1 above)
Comment (trwalke) [reply tests/Microsoft.Identity.Web.Test.Common/TestConstants.cs:115]: Some of the tests here use this secret in MSAL directly to authenticate with an app. This isnt for lab access, but for testing a client secret flow. IdWeb uses the lab api to acquire this secret by name from the lab keyvault

File: tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs (hunk 2 above)
Comment (trwalke) [reply tests/Microsoft.Identity.Web.Test.Integration/AcquireTokenForAppIntegrationTests.cs:52]: This is where IdWeb uses the Lab api to acquire secrets from the Lab Keyvault to run client credential scenarios. This is how it acquires the "MSIDLAB4-IDLABS-APP-AzureADMyOrg-CC" secret.

**Review by jennyf19 — APPROVED** [review]
